migration/sql-files/078-assignee-and-label-empty-value.sql

```diff
@@ -0,0 +1,5 @@
+-- Remove 'system.assignees' from the fields if the value is 'null' or '[]'
+update work_items set fields=fields - 'system.assignees' where Fields->>'system.assignees' is null;
```

Comment:
Can you use a placeholder instead of `system.assignees` maybe? Then pass in the `workitem.SystemAssignees` constant from code.

Reply:
Fixed. 9392cc941712846c91f70d9a694da3a5c30acb4a